src/text.lisp

```diff
@@ -1,5 +1,29 @@
 (in-package :trivial-clipboard)
 
+(defvar *clipboard-in-command*
+  #+(or darwin macosx)
+  "pbcopy"
+  #+(and :unix (:not :darwin))
+  "xclip")
+
```

Comment:
I prefer ``*clipboard-in-command*``  being a list that include ``*clipboard-in-args*`` information.

Reply:
I would recommend against it because a separate variable makes it easy for packagers /users to patch/set the path to the executable, without having to parse a Lisp list (in case of a patch) and without having to bother about the arguments.

Reply:
I'll consider it when packager/user claim about it.

Reply:
ok :(